ctera/src/main/java/ch/cyberduck/core/ctera/CteraListService.java

```diff
@@ -0,0 +1,58 @@
+package ch.cyberduck.core.ctera;
+
+/*
+ * Copyright (c) 2002-2024 iterate GmbH. All rights reserved.
+ * https://cyberduck.io/
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ */
+
+import ch.cyberduck.core.Acl;
+import ch.cyberduck.core.Path;
+import ch.cyberduck.core.dav.DAVListService;
+import ch.cyberduck.core.dav.DAVPathEncoder;
+import ch.cyberduck.core.exception.BackgroundException;
+
+import javax.xml.namespace.QName;
+import java.io.IOException;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import com.github.sardine.DavResource;
+
+import static ch.cyberduck.core.ctera.CteraAclPermissionFeature.*;
+
+public class CteraListService extends DAVListService {
+
+    private final CteraSession session;
+
+    public CteraListService(final CteraSession session) {
+        super(session, new CteraAttributesFinderFeature(session));
+        this.session = session;
+    }
+
+    @Override
+    protected List<DavResource> list(final Path directory) throws IOException, BackgroundException {
+        return session.getClient().list(new DAVPathEncoder().encode(directory), 1, Collections.unmodifiableSet(Stream.concat(
+                // N.B. Timestamp feature disabled in CteraSession.getFeature(Timestamp.class)
+                Stream.of(new QName(CTERA_NAMESPACE_URI, CteraAttributesFinderFeature.CTERA_GUID, CTERA_NAMESPACE_PREFIX)),
+                allCteraCustomACLQn.stream()
+        ).collect(Collectors.toSet())));
+    }
+
+    public void preflight(final Path directory) throws BackgroundException {
```

Comment:
Remove this preflight check with no interface.

Reply:
1a32ebe466b9b3b6caac80a8d0f8aeb272718965